samples/templates-noaci/container_prep.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Can we use bash here?

Reply:
The files under the "samples" directory are very specific to configuring an OSP install with Director. I have delivered them as I received them. @homeski is the expert on that, but they should probably be left as-is (same for the other files you commented on in this directory).

Reply:
I figured as much, and I agree that they should be left as-is for the reasons given.  Just wanted to check.